scripts/assets/kdoc_validity_exemptions.textproto

```diff
@@ -92,9 +92,9 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/home/RouteToTopicPl
 exempted_file_path: "app/src/main/java/org/oppia/android/app/home/promotedlist/ComingSoonTopicsViewModel.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/home/promotedlist/PromotedStoryListViewModel.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModel.kt"
-exempted_file_path: "app/src/main/java/org/oppia/android/app/home/recentlyplayed/OngoingListAdapter.kt"
-exempted_file_path: "app/src/main/java/org/oppia/android/app/home/recentlyplayed/OngoingStoryClickListener.kt"
-exempted_file_path: "app/src/main/java/org/oppia/android/app/home/recentlyplayed/OngoingStoryViewModel.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedStoryClickListener.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedStoryListAdapter.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedStoryViewModel.kt"
```

Comment:
I think that it'd be nice, while here, to go ahead and finish up the KDocs for these files so that we don't need to continue carrying the exemptions.

Reply:
I have added the KDocs and removed these excemptions.